test/awss3.jl

```diff
@@ -126,6 +126,10 @@ end
     @test (s3_get(aws, bucket_name, "key3"; version=versions[3]["VersionId"]) == b"data3.v1")
     @test (s3_get(aws, bucket_name, "key3"; version=versions[2]["VersionId"]) == b"data3.v2")
     @test (s3_get(aws, bucket_name, "key3"; version=versions[1]["VersionId"]) == b"data3.v3")
+
+    tmp_file = "/tmp/jl_qws_test_key3"
```

Comment:
Avoid making assumptions about the filesystem:
```suggestion
    tmp_file = joinpath(tempdir(), "jl_qws_test_key3")
```